packages/optimizer/README.md

```diff
@@ -205,6 +222,13 @@ const ampOptimizer = AmpOptimizer.create({
 - default: `{}`
 - used by: [AutoExtensionImport](lib/transformers/AutoExtensionImporter.js)
 
+#### `fetch`
+
+Specifies the fetch implementation to use:
```

Comment:
nit, i'd link to something about what `fetch` is (the [spec](https://fetch.spec.whatwg.org/) maybe?)